helpdesk-service/helpdesk-service-client/src/main/java/org/symphonyoss/symphony/bots/helpdesk/service/ticket/client/TicketClient.java

```diff
@@ -123,6 +125,25 @@ public Ticket getUnresolvedTicketByClientStreamId(String clientStreamId) {
     }
   }
 
+  /**
+   * Gets a unresolved ticket list
+   * @return the ticket
+   */
+  public List<Ticket> getUnresolvedTickets() {
+    try {
+      List<Ticket> ticketList = ticketApi.searchTicket(groupId, null, null);
+
+      if ((ticketList != null) && (!ticketList.isEmpty())) {
+        return ticketList.stream()
+            .filter((ticket) -> !ticket.getState().equals(TicketStateType.RESOLVED.getState()))
+            .collect(Collectors.toList());
+      }
+      return null;
```

Comment:
Returns an empty list instead of a null value.

Reply:
ok